src/sqlancer/hsqldb/ast/HSQLDBUnaryPrefixOperation.java

```diff
@@ -0,0 +1,83 @@
+package sqlancer.hsqldb.ast;
+
+import sqlancer.Randomly;
+import sqlancer.common.ast.BinaryOperatorNode;
+import sqlancer.common.ast.newast.NewUnaryPrefixOperatorNode;
+import sqlancer.common.ast.newast.Node;
+import sqlancer.hsqldb.HSQLDBSchema;
+
+public class HSQLDBUnaryPrefixOperation extends NewUnaryPrefixOperatorNode<HSQLDBExpression> {
+
+    // private final HSQLDBUnaryPrefixOperation.HSQLDBUnaryPrefixOperator operation;
+    // private final Node<HSQLDBExpression> expression;
```

Comment:
Perhaps you can delete the comments in the next PR?

Reply:
Sure , will do